src/plugins/ui_plugin.ts

```diff
@@ -25,8 +25,19 @@ export interface UIPluginConfig {
   readonly session: Session;
 }
 
-export interface UIPluginConstructor {
-  new (config: UIPluginConfig): UIPlugin;
+export interface CoreViewPluginConfig extends CommonUIPluginConfig {
+  readonly getters: CoreViewGetters;
+  readonly allGetters: Getters;
```

Comment:
EvaluationPlugin needs all getters to give to the functions for their evaluation (eg. odoo list functions)  :x